Source/Krypton Components/Krypton.Navigator/Controller/DragViewController.cs

```diff
@@ -314,15 +314,15 @@ public virtual bool KeyUp([DisallowNull] Control c, [DisallowNull] KeyEventArgs
         /// Source control has lost the focus.
         /// </summary>
         /// <param name="c">Reference to the source control instance.</param>
-        public virtual void GotFocus(Control c)
+        public virtual void GotFocus(Control? c)
         {
         }
 
         /// <summary>
         /// Source control has lost the focus.
         /// </summary>
         /// <param name="c">Reference to the source control instance.</param>
-        public virtual void LostFocus([DisallowNull] Control c)
+        public virtual void LostFocus([DisallowNull] Control? c)
```

Comment:
Please undo these nullables
It specifies not null, and now you are allowing null.
Also the warning has jump by nearly 20k so this has gone in the wrong direction.

Reply:
Reverted, however some of the ribbon/docking/workspace examples currently crash